app/atomic_api.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from aiohttp import web
+from aiohttp_jinja2 import template
+
+from app.utility.base_service import BaseService
+from app.service.auth_svc import for_all_public_methods, check_authorization
+from plugins.atomic.app.atomic_svc import AtomicService
+
+
+@for_all_public_methods(check_authorization)
+class AtomicAPI(BaseService):
+
+    def __init__(self, services, name, description):
+        self.services = services
+        self.auth_svc = self.services.get('auth_svc')
+        self.data_svc = self.services.get('data_svc')
+        self.name = name
+        self.description = description
+        self.atomic_svc = AtomicService()
```

Comment:
`atomic_svc` doesn't appear to be used.
```suggestion
        self.atomic_svc = AtomicService()
```

Reply:
Also as an aside, my instinct is to avoid initializing dependencies directly within your method and lean more towards passing them in if you can. This can make testing a bit easier and decouple these two components a bit ("new is glue" n' all that).